### PR TITLE
LEB-1923 Drupal 7 Language Context Implementation

### PR DESCRIPTION
**Pull request by nkarpenko**
The D7 implementation of [Language Capture for D8](https://github.com/acquia/acquia_lift/pull/272/commits/c15cbf0585576ee8914f8d5451f1958101b9f841). Also contains the unit testing file which was successful.

### CODE REVIEW

File: acquia_lift.context.inc [hunk 1]
```diff
@@ -75,9 +75,14 @@ function _acquia_lift_get_page_context($node = NULL, $primary = FALSE) {
...
   // Add the content type of the page if this is the primary node displayed.
   if ($primary && empty($page_context['post_id'])) {
+    // Get global language to set as 'context_langauge'
+    // Note: Language will always be set as there is a default fallback
+    global $language;
+
     $page_context['content_type'] = $node->type;
     $page_context['content_title'] = $node->title;
     $page_context['published_date'] = $node->created;
+    $page_context['context_language'] = $language->language;
```
Comment (ynx) [inline acquia_lift.context.inc]: New code on language should probably be outside of this if statement, so `context_language` is surfaced to markup on non-node pages (e.g. home page).

File: acquia_lift.context.inc [hunk 2]
```diff
@@ -75,9 +75,14 @@ function _acquia_lift_get_page_context($node = NULL, $primary = FALSE) {
 
   // Add the content type of the page if this is the primary node displayed.
   if ($primary && empty($page_context['post_id'])) {
+    // Get global language to set as 'context_langauge'
```
Comment (ynx) [inline acquia_lift.context.inc]: `context_langauge` typo.

File: tests/acquia_lift.test [hunk 3]
```diff
@@ -483,6 +483,7 @@ class AcquiaLiftsWebTest extends DrupalWebTestCase {
 
     $this->drupalGet('node/' . $post_id);
     $this->assertLiftMetaTag('content_type', 'article');
+    $this->assertLiftMetaTag('context_language', 'en');
```
Comment (ynx) [inline tests/acquia_lift.test:492]: I would use `fr` instead of `en` because `en` is Drupal default.

Comment (ynx) [reply tests/acquia_lift.test:492]: `testNonNodePage` needs to assert `context_language` as well.

**Review by ynx — APPROVED** [review]
I have not tried feature personally but code reads great for me. Left some minor comments but non blocking.

File: tests/acquia_lift.test (hunk 3 above)
Comment (nkarpenko) [reply tests/acquia_lift.test:492]: As discussed prior, this is tested functionality in D7, grabbing the current language so only en works as an assertion. Added to test node page

File: acquia_lift.context.inc (hunk 1 above)
Comment (nkarpenko) [reply acquia_lift.context.inc]: added outside of statement

File: acquia_lift.context.inc (hunk 2 above)
Comment (nkarpenko) [reply acquia_lift.context.inc]: fixed

**Review by ynx — APPROVED** [review]

**Pull request merged**